### PR TITLE
Other rejection reasons

### PR DESCRIPTION
**Pull request by rajatsajwan**
Targets #2683

### Description
Other rejection reasons option. 

### Checklist:

- [ ] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by rajatsajwan** [comment]
Screenshot_20230109_101121
sir I added other option

**Comment by rajatsajwan** [comment]
Screenshot (46)
it is saved in data base.

**Comment by rajatsajwan** [comment]
yes , sir i tested the functionality.

File: Modules/HR/Http/Requests/Recruitment/ApplicationRoundRequest.php [hunk 1]
```diff
@@ -45,21 +45,14 @@ public function rules()
...
             if (request()->input('send_mail_to_applicant.confirm') == 'on') {
                 $rules['mail_to_applicant.confirm.subject'] = 'nullable|string|required_with:send_mail_to_applicant';
                 $rules['mail_to_applicant.confirm.body'] = 'nullable|string|required_with:send_mail_to_applicant';
             }
-        } elseif (request()->input('action') == 'reject') {
-            $rules['reject_reason'] = 'nullable|array';
-            if (request()->input('send_mail_to_applicant.reject') == 'on') {
-                $rules['mail_to_applicant.reject.subject'] = 'nullable|string|required_with:send_mail_to_applicant';
-                $rules['mail_to_applicant.reject.body'] = 'nullable|string|required_with:send_mail_to_applicant';
-            }
```
Comment (P4NK4J) [inline Modules/HR/Http/Requests/Recruitment/ApplicationRoundRequest.php:60]: Why this code is removed? What would happen is reject action is selected ?

**Review by P4NK4J — COMMENTED** [review]
Left a question, please take a look.

File: Modules/HR/Http/Requests/Recruitment/ApplicationRoundRequest.php (hunk 1 above)
Comment (rajatsajwan) [reply Modules/HR/Http/Requests/Recruitment/ApplicationRoundRequest.php:60]: Screenshot (47)

**Comment by rajatsajwan** [comment]
Screenshot (48)

File: Modules/HR/Http/Requests/Recruitment/ApplicationRoundRequest.php (hunk 1 above)
Comment (rajatsajwan) [reply Modules/HR/Http/Requests/Recruitment/ApplicationRoundRequest.php]: sir this code required subject & body to run ...

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Pull request merged**